lib/ReactGridLayout.jsx

```diff
@@ -315,7 +315,7 @@ export default class ReactGridLayout extends React.Component<Props, State> {
     // Legacy support for compactType
     // Allow parent to set layout directly.
     if (
-      !isEqual(nextProps.layout, prevState.propsLayout) ||
+      !isEqual(nextProps.layout, prevState.layout) ||
```

Comment:
we can't compare layout from props and from state here because of `synchronizeLayoutWithChildren`, which will create `synchronized` layout (with different fields) so `isEqual(nextProps.layout, prevState.layout)` will be always false. In this case we want to compare new layout from props and the previous one. And it work now pretty good

I tried to figure out what is the problem here https://github.com/STRML/react-grid-layout/issues/1011 but I couldn't

Could you describe the issue with an example please?